outline.md

```diff
@@ -343,34 +335,164 @@ This creates a few files, and also adds a  `app/router.js`:
 
     export default Router;
 
-TODO: Should we add a store lookup to our route.js file even though it's done automagically?
+Here it has defined a resource for us with a dynamic segmant in the route, `:blog_post_id`. This dynamic segment will be extracted from the URL and passed into the `model` hook on the `post` route. We can then use this parameter to look up that exact `blog post` in the store. So let's open up `routes/blog-post.js` that was generated for us and do just that.
+
+    import Ember from 'ember';
+
+    export default Ember.Route.extend({
+      model: function(params) {
+        return this.store.find('blog-post', params.blog_post_id);
+      }
+    });
+
 
 #### Update the template
 
-Now let's add the following to our `app/blog-post.hbs` template file to display our post on the page:
+In order to make sure this is working, let's add some markup to `app/blog-post.hbs` that will display a post:
 
     <article>
       <h2>{{model.title}}</h2>
       {{model.body}}
     </article>
 
-If we visit `http://localhost:4200/post/1` in our browser, we should see an example blog post.
+Since we happen to know there is a blog post with `id: 1` on our API server, we can manually visit `http://localhost:4200/post/1` in our browser to test with an example blog post.
+
+#### The magic of Ember-Data
```

Comment:
Should this section replace the API table/explanation in the section [8 Diversion: Accessing our API with ember-data](https://github.com/hbrysiewicz/ember-cli-101/blob/master/outline.md#8-diversion-accessing-our-api-with-ember-data)?